DESCRIPTION

```diff
@@ -3,7 +3,7 @@ Title: Easy Access To `ggplot2` Commands
 Version: 0.1.0
 Authors@R: c(
  person("Jonathan", "Carroll", email = "rpkg@jcarroll.com.au", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-1404-5264")),
- person("Alicia", "Schep", email = "aschep@stanford.edu", role = c("aut"))
+ person("Alicia", "Schep", email = "aschep@gmail.com", role = c("aut"))
```

Comment:
Do you have an ORCID to add sometime? They now show up with icons on the CRAN page (that is going to be their 'unique' identifier.

Reply:
Yeah I have an ORCID.  I'll track it down and add.